templates/index.html

```diff
@@ -43,7 +43,7 @@
       }
     }
     function showPrs() {
-      var username = document.querySelector('.js-username').value;
+      var username = document.querySelector('.js-username').value.toLowerCase();
```

Comment:
Like I said in the comment above, we should preserve the case sent by the user and only manipulate it for cache purposes and not everywhere.